### PR TITLE
Warp song text now shows the correct destination

### PR DESCRIPTION
**Pull request by matthewkirby**
If warp song shuffle was enabled, playing a warp song asked if you wanted to "Warp to {vanilla destination}". It is now updated to reflect the destination.

See it in action: https://www.youtube.com/watch?v=2e31mAZUIWg

### CODE REVIEW

**Comment by r0bd0g** [comment]
One awkward thing about this is that the vanilla text for minuet is "Lost Woods" when it's really Sacred Forest Meadow.

**Comment by matthewkirby** [comment]
That is true. I could pretty easily make SFM give you lost woods text since everything is already converted from raw region name to a pretty looking destination. But I think that SFM would be better because if overworld ER is on, they aren't necessarily connected.

**Comment by fenhl** [comment]
I think it should say SFM as “Lost Woods” is just wrong in vanilla. It might even make sense to just correct that even when warps aren't shuffled, similar to how rando fixes the “Keatan” typo.

**Comment by fenhl** [comment]
If I'm reading this right, this generates text like “Warp to Deku Tree?”, so it's missing a “the” or “a” in front of some region names. Also ReDead and Dampé are misspelled.

**Comment by fenhl** [comment]
I also think that with misc. hints off, this should say something like “Warp to a mysterious place?” instead.

**Comment by matthewkirby** [comment]
> If I'm reading this right, this generates text like “Warp to Deku Tree?”, so it's missing a “the” or “a” in front of some region names. Also ReDead and Dampé are misspelled.

Oh true. I can correct those. 

> I also think that with misc. hints off, this should say something like “Warp to a mysterious place?” instead.
I think I disagree with this. TBH, I don't really like that we are tieing other things into the altar hint. If its with misc hints and you decide you dont want this text then you no longer get the altar text. I would honestly prefer this to just be always on (like this PR) or always off (change everything to just say "mysterious land"). If the devs want me to tie it to that setting, fine, but I am going to wait on their call for that one.

**Comment by r0bd0g** [comment]
Isn't the vanilla text just "Warp to Lost Woods"? I can't remember if it says 'the' at all.

**Comment by matthewkirby** [comment]
Does it need to identically match?

**Comment by Zannick** [comment]
I'm going to make the call that this shouldn't be considered a hint, so leave it as accurately telling where it's going is good.

File: Messages.py [hunk 1]
```diff
@@ -993,3 +1084,35 @@ def shuffle_group(group):
...
+        for formatted_region, info in REGION_NAMES.items():
+            raw_region_list, region_color = info
+            if destination_raw in raw_region_list:
+                destination = formatted_region
+                color = region_color
+                break
+
+        # Check if region name isn't found
+        if destination is None:
+            print("MISSING REGION NAME: " + destination_raw)
```
Comment (Zannick) [inline Messages.py:1113]: nit: print should be logging.error

**Review by Zannick — APPROVED** [review]

**Comment by r0bd0g** [comment]
Is this text excluded in 'except hints' text shuffle? (Did I ask this already?)

File: Messages.py (hunk 1 above)
Comment (matthewkirby) [reply Messages.py]: Done

**Comment by Zannick** [comment]
> Is this text excluded from text shuffle when text is shuffled 'except hints'? (Did I ask this already?)

Does not appear so. Should it? Are these messages grouped together under the "goto" category? I'm going to merge without it for now, and we can have a follow up PR to fix it.

**Comment by r0bd0g** [comment]
Honestly I'm not sure whether or not that text should be shuffled. It's not really needed since you can just warp to see where it goes. Similar to shop chus where you have to try to buy them to see how much they cost.

**Pull request merged**